app/uk/gov/hmrc/agentclientrelationships/auth/AuthActions.scala

```diff
@@ -80,6 +81,39 @@ trait AuthActions extends AuthorisedFunctions {
   def hasRequiredStrideRole(enrolments: Enrolments, strideRoles: Seq[String]): Boolean =
     strideRoles.exists(s => enrolments.enrolments.exists(_.key == s))
 
+  def determineService(enrolment: Enrolment): Either[Unit, TypeOfEnrolment] =
```

Comment:
this function can be moved into `TypeOfEnrolment` , it could be simply 
`def getByEnrolmentKey(...)`

Reply:
ok